package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@influxdata/clockface",
-  "version": "3.6.0",
+  "version": "3.6.1",
```

Comment:
I think this should technically be `3.7.0` since it's adding new, non-breaking functionality